build.sh

```
@@ -388,11 +388,11 @@ if buildAll || hasArg libcuopt || hasArg cuopt_grpc_server; then
     if hasArg cuopt_grpc_server && ! hasArg libcuopt && ! buildAll; then
         # Build only the gRPC server (ninja resolves libcuopt as a dependency)
         cmake --build "${LIBCUOPT_BUILD_DIR}" --target cuopt_grpc_server ${VERBOSE_FLAG} ${JFLAG}
-    elif hasArg -n; then
+    elif hasArg --install; then
+        cmake --build "${LIBCUOPT_BUILD_DIR}" --target ${INSTALL_TARGET} ${VERBOSE_FLAG} ${JFLAG}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)